solo-leveling-telegram-bot/src/main/kotlin/com/sleepkqq/sololeveling/telegram/bot/service/broadcast/impl/BroadcastRunner.kt

```
@@ -0,0 +1,83 @@
+
+				currentPage++
+			} while (response.paging.hasMore)
+
+			status = BroadcastStatus.COMPLETED
+
+			log.info(
+				"Broadcast {} completed: total={} success={}, failed={}",
+				broadcast.id, total, totalSuccess, totalFailed
+			)
+
+		} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)